port/unix/omrsysinfo.c

```diff
@@ -253,6 +253,18 @@ static intptr_t omrsysinfo_get_aix_ppc_description(struct OMRPortLibrary *portLi
 #define __power_10() (_system_configuration.implementation == POWER_10)
 #endif /* !defined(__power_10) */
 
+#if !defined(__power_11)
+#define POWER_11 0x80000 /* Power 11 class CPU */
+#define __power_11() (_system_configuration.implementation == POWER_11)
+#endif /* !defined(__power_11) */
```

Comment:
I don't see this code being used (dead code). What is the intent of adding it?

Reply:
it is put there for anybody to see how __power_latestKnownAndUp comes by and how to update when pFuture is supported. also for what to do, when we need to support pFuture (i.e. we need to do  `else if (__power_11()) ... else if (__power_latestKnownAndUp()) ...`